app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Good job so far 👏 👏 However, from the project requirements, you are expected to add `:confirmable` attribute for the user model. This means that the user will receive a confirmation email for authentication 😃 👍